src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
Avoid using magic numbers directly in your code. Here, '100' is a magic number. Define it as a constant at the top of your class with a clear name indicating its purpose.